kafka/consumer/group.py

```diff
@@ -181,6 +182,16 @@ def __init__(self, *topics, **configs):
                         new_config, self.config['auto_offset_reset'])
             self.config['auto_offset_reset'] = new_config
 
+        metrics_tags = {'client-id': self.config['client_id']}
+        metric_config = MetricConfig(samples=self.config['metrics_num_samples'],
+                                     time_window_ms=self.config['metrics_sample_window_ms'],
+                                     tags=metrics_tags)
+        reporters = [reporter() for reporter in self.config['metric_reporters']]
+        reporters.append(DictReporter('kafka.consumer'))
```

Comment:
i thought about it. i think the idea is to always guarantee that there is this 'default' reporter regardless of what is configured. probably harmless to move to a default config, but i thought i'd leave it 1:1 with the java implementation until we get more things instrumented and have a better feel for things.